ansible_anonymizer/anonymizer.py

```diff
@@ -85,6 +85,18 @@ def is_password_field_name(name: str) -> bool:
     return re.search(DENYLIST_REGEX_WITH_PREFIX, name) is not None
 
 
+def is_jinja2_expression(value: str, quoted: bool = True) -> bool:
+    if quoted:
+        if re.match(r"^\s*\"\s*{{\s*.*?\s*}}\s*\"\s*$", value):
```

Comment:
Though I am not an expert of regex and it won't make any differences, isn't `{{\s*.*?\s*}}` equivalent to `{{.*?}}`?

Reply:
Indeed, the `\s` are superfluous. I will right this differently.